src/game/Spell.cpp

```diff
@@ -4193,6 +4193,9 @@ SpellCastResult Spell::CheckCast(bool strict)
             return SPELL_FAILED_NOT_READY;
     }
 
+    if (!m_caster->isAlive() && !m_spellInfo->HasAttribute(SPELL_ATTR_CASTABLE_WHILE_DEAD))
```

Comment:
The answer lies in the first PR comment :) 

https://github.com/killerwife/mangos-tbc/commit/19e63b82b716f73e1fa7e83f7046034adcefe9cc

You started working on this, and you didnt finish it. If you finish it. it will be defined :)